### PR TITLE
Switch from Carnot to Cryptarchia

### PR DESCRIPTION
**Pull request by zeegomo**
Remove Carnot related stuff and use crytparchia as the consensus protocol for our node

### CODE REVIEW

**Review by danielSanchezQ — APPROVED** [review]
-12k, Flying light!!

File: tests/src/lib.rs [hunk 1]
```diff
@@ -58,20 +98,41 @@ impl SpawnConfig {
         Self::Chain {
             consensus: ConsensusConfig {
                 n_participants,
-                // All nodes are expected to be responsive in happy-path tests.
-                threshold: Fraction::one(),
-                // Set the timeout conservatively
-                // since nodes should be spawned sequentially in the chain topology
-                // and it takes 1+ secs for each nomos-node to be started.
-                timeout: adjust_timeout(Duration::from_millis(n_participants as u64 * 2500)),
+                // by setting the slot coeff to 1, we also increase the probability of multiple blocks (forks)
```
Comment (davidrusu) [inline tests/src/lib.rs:101]:
```suggestion
                // by setting the slot coeff close to 1, we also increase the probability of multiple blocks (forks)
```

**Review by davidrusu — APPROVED** [review]
🔪 that's a lot of red!

**Review by youngjoon-lee — APPROVED** [review]
The integration test will pass if we increase `emission_rate_per_min` (for example to 500). I'm taking a look at why the test fails with the current setting (why the `CommitmentNotFound` error occurs).

File: tests/src/lib.rs (hunk 1 above)
Comment (zeegomo) [reply tests/src/lib.rs:101]: thanks, forgot to update it

**Pull request merged**